blueocean-core-js/src/less/run-button.less

```diff
@@ -37,6 +37,14 @@
                 position: absolute;
                 top: 0;
                 left: 0;
+
+                .ParametersRunButton & {
+                    fill: rgba(53, 64, 82, 0.25);
```

Comment:
Where do these values come from?

Reply:
I added the same colours as the history button that is beside it, it gets them from here https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-dashboard/src/main/less/history-button.less#L14

Reply:
👍